index.html

```diff
@@ -4833,6 +4662,183 @@ <h5>Note regarding the ARIA 1.3 <code>image</code> role.</h5>
+			<table class="role-features">
+			  <caption>Characteristics:</caption>
+			  <thead>
+				<tr>
+				  <th scope="col">Characteristic</th>
+				  <th scope="col">Value</th>
+				</tr>
+			  </thead>
+			  <tbody>
+				<tr>
+				  <th class="role-abstract-head" scope="row">Is Abstract:</th>
+				  <td class="role-abstract"> </td>
+				</tr>
+				<tr>
+				  <th class="role-parent-head" scope="row">Superclass Role:</th>
+				  <td class="role-parent"><rref>section</rref></td>
+				</tr>
+				<tr>
+				  <th class="role-children-head" scope="row">Subclass Roles:</th>
+				  <td class="role-children">Placeholder</td>
+				</tr>
+				<tr>
+				  <th class="role-base-head" scope="row">Base Concept:</th>
+				  <td class="role-base"><a href="https://www.w3.org/TR/html5/grouping-content.html#the-dt-element"><abbr title="Hypertext Markup Language">HTML</abbr> <code>dt</code></a></td>
+				</tr>
+				<tr>
+				  <th class="role-related-head" scope="row">Related Concepts:</th>
+				  <td class="role-related"> </td>
+				</tr>
+				<tr>
+				  <th class="role-scope-head" scope="row">Required Context Role:</th>
+				  <td class="role-scope"><rref>list</rref></td>
+				</tr>
+				<tr>
+				  <th class="role-required-properties-head">Required States and Properties:</th>
+				  <td class="role-required-properties"> </td>
+				</tr>
+				<tr>
+				  <th class="role-properties-head" scope="row">Supported States and Properties:</th>
+				  <td class="role-properties">
+					<ul>
+					  <li><pref>aria-level</pref></li>
+					  <li><pref>aria-posinset</pref></li>
+					  <li><pref>aria-setsize</pref></li>
+					</ul>
+				  </td>
+				</tr>
+				<tr>
+				  <th class="role-inherited-head" scope="row">Inherited States and Properties:</th>
+				  <td class="role-inherited">Placeholder</td>
+				</tr>
+				<tr>
+				  <th class="role-namefrom-head" scope="row">Name From:</th>
+				  <td class="role-namefrom">author</td>
+				</tr>
+				<tr>
+				  <th class="role-namerequired-head" scope="row">Accessible Name Required:</th>
+				  <td class="role-namerequired"> </td>
+				</tr>
+				<tr>
+				  <th class="role-namerequired-inherited-head" scope="row">Inherits Name Required:</th>
+				  <td class="role-namerequired-inherited"> </td>
+				</tr>
+				<tr>
+				  <th class="role-childpresentational-head" scope="row">Children Presentational:</th>
+				  <td class="role-childpresentational"> </td>
+				</tr>
+				<tr>
+				  <th class="role-presentational-inherited-head" scope="row">Inherits Presentational:</th>
+				  <td class="role-presentational-inherited"> </td>
+				</tr>
+			  </tbody>
+			</table>
+		  </div>
+		  <div class="role" id="listitemvalue">
+			<rdef>listitemvalue</rdef>
+			<div class="role-description">
+			  <p>A single value item in a list.</p>
+	  
+			  <p>Author requirements for <a>elements</a> whose <a>role</a> is <code>listitemvalue</code>:</p>
+			  <ul>
+				<li>MUST be contained in an element whose <a>role</a> is <rref>list</rref>.</li>
```

Comment:
repeat of previous comment:
we should probably mention here that there should (must?) be no intervening listitem elements that interrupt the expected sibling relationship to the listitemkey/listitemvalue